language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/CommandExecutor.java

```diff
@@ -166,7 +173,8 @@ private static Object executeImportPackage(LSServiceOperationContext context) th
 
             if (endCol != -1) {
                 int contentLengthToReplaceStart = fileContent.substring(0,
-                        fileContent.indexOf(contentComponents[endLine])).length() + endCol + 1;
+                                                                        fileContent.indexOf(contentComponents[endLine]))
+                        .length() + endCol + 1;
```

Comment:
Shall we reformat this line?

Reply:
Fixed it, the whole class is accidentally formatted incorrectly.